packages/server/src/service/pdfForm/westVirginia.ts

```diff
@@ -0,0 +1,140 @@
+    text(birthYear[0], 0, 547, 154)
+    text(birthYear[1], 0, 557, 154)
+    text(birthYear[2], 0, 567, 154)
+    text(birthYear[3], 0, 577, 154)
+
+    // 'in the city/county of:' (locale)
+    text(stateInfo.county ? stateInfo.county : '', 0, 460, 137)
+
+
+    // Street and street number.
+    const streetAddress = stateInfo.address.streetNumber + ' ' + stateInfo.address.street + (stateInfo.address.unit ? ',' : '')
+    const streetAddressStart = 195
+    text(streetAddress, 0, streetAddressStart, 137)
+    // Apt/suite
+    const addressPosition = streetAddress.length * 7 + streetAddressStart
+    text(stateInfo.address.unit ? stateInfo.address.unit : '', 0, addressPosition, 137)
+    // City
+    text(stateInfo.address.city ? stateInfo.address.city : ' ', 0, 130, 154)
+    // Zip code.
+    text(stateInfo.address.postcode[0], 0, 367, 154)
+    text(stateInfo.address.postcode[1], 0, 377, 154)
+    text(stateInfo.address.postcode[2], 0, 387, 154)
+    text(stateInfo.address.postcode[3], 0, 397, 154)
+    text(stateInfo.address.postcode[4], 0, 407, 154)
+
+    // Mailing address.
+    // TODO: This s currently operating on the assumption that the mailing and regular address
+    // are the same. This is likely an unsafe assumption. mailingAddress should also be 
+    // an object of type address in stateInfo
+    const mailingAddressStart = 150
+    text(streetAddress, 0, mailingAddressStart, 175)
+    // Apt/suite
+    const mailingPosition = streetAddress.length * 7 + mailingAddressStart
+    text(stateInfo.address.unit ? stateInfo.address.unit : '', 0, mailingPosition, 175)
+    // City
+     text(stateInfo.address.city ? stateInfo.address.city : ' ', 0, 130, 192)
+    // state
+    text("WV", 0, 295, 192)
+    // Zip code.
+    text(stateInfo.address.postcode[0], 0, 367, 192)
+    text(stateInfo.address.postcode[1], 0, 377, 192)
+    text(stateInfo.address.postcode[2], 0, 387, 192)
+    text(stateInfo.address.postcode[3], 0, 397, 192)
+    text(stateInfo.address.postcode[4], 0, 407, 192)
+
+    // Phone number.
+    const phoneNumber = cleanPhoneNumber(stateInfo.phone)
+    text(phoneNumber[0], 0, 460, 175)
+    text(phoneNumber[1], 0, 467, 175)
+    text(phoneNumber[2], 0, 474, 175)
+    text(phoneNumber[3], 0, 481, 175)
+    text(phoneNumber[4], 0, 488, 175)
+    text(phoneNumber[5], 0, 495, 175)
+    text(phoneNumber[6], 0, 502, 175)
+    text(phoneNumber[7], 0, 509, 175)
+    text(phoneNumber[8], 0, 516, 175)
+    text(phoneNumber[9], 0, 523, 175)
+
+
+    // Reason for election
+    check(0, 130, 218)
+
+    // election
+    if (stateInfo.election){
+        if (stateInfo.election == 'Federal/State/County') {
+            check(0, 114, 480)
+        } else {
+            check(0, 114, 492)
+        }
+
+    } else {
+        check(0, 114, 480)
+    }
+
+    if (stateInfo.electionType){
+        if (stateInfo.electionType == 'Primary') {
+            check(0, 210, 491)
+
+
+        } else if (stateInfo.electionType == 'General') {
```

Comment:
Shouldn't this be for Primary?

Reply:
I think it might be good idea to add a comment above each check saying what the voter is going for, if that makes sense.  It can be inferred by tracing the conditionals, but it would make it more easier to read. e.g.: 

`// Voting in the state/federal/county primary as a Democrat`

, or something.

Reply:
Yep. Added comments as well.